Makefile

```diff
@@ -41,16 +42,6 @@ tool-install:
 		gotest.tools/gotestsum \
 		github.com/rhysd/actionlint/cmd/actionlint
 
-buf: buf-web
-
-buf-web: tool-install
```

Comment:
reminder to self, will need to add sessions proto to sdk instead of using buf here